source/Calamari.Tests/ArgoCD/Git/TestGitConnection.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace Calamari.Tests.ArgoCD.Git
 {
-    public class GitConnection : IGitConnection
+    public class TestGitConnection : IGitConnection
```

Comment:
is this still needed, or should we just use the GitConnection struct from the prod code?